eisbuk-admin/src/translations/en.json

```diff
@@ -129,5 +130,11 @@
     "CancelSlot": "Cancel slot creation",
     "ConfirmCreateSlot": "Confirm slot creation",
     "ConfirmEditSlot": "Confirm slot edits"
+  },
+  "Title": {
+    "UpcomingBirthdays": "Upcoming Birthdays"
+  },
+  "MenuButton": {
+    "ShowAll": "Show All"
```

Comment:
This menu button says "show all". I can understand it by reading this file.

But the context is not clear: does this "show all" display on bookings? I can't really tell by just watching here.
I propose we change the position of these labels a bit: we can introduce a new section `BirthdaysMenu` to host the two strings that we need. Then we can move the two existing translations inside it.

Something like this:

```json
{
    "BirthdayMenu": {
        "ShowAll": "Show All",
        "UpcomingBirthdays": "Upcoming Birthdays"
    }
}
```

It's a bit of a tedious change, since it requires updating both translations and the translations enums, but I think it's worth it.

Reply:
To clarify: in Italian we might need different translations for `Show All` depending on the gender of the "thing" in `all`. In Italian things have a gender...
So we might need two (potentially more) different `Show All` translatable strings...

Reply:
Well, in this case it needs to say `Show All [Birthdays/Items]` so I changed the Italian to "Mostra Tutti". Is this correct?

Reply:
Yep! It's the kind of difference I meant. In this case _tutti_ is slightly better than _tutto_.